packages/lit-starter-ts/tsconfig.json

```diff
@@ -26,7 +26,8 @@
         "name": "ts-lit-plugin",
         "strict": true
       }
-    ]
+    ],
+    "types": ["mocha"]
```

Comment:
Are explicit types required on starter packages?

Reply:
It might not be needed when it's cloned to its own template repo but it's needed for the build and test to run within the monorepo.